### DISCUSSION TITLE
Can i generate IObservable instead of Task?

### DISCUSSION THREAD

**Discussion by janfolbrecht** [christianhelle/refitter#321] [Q&A] [answered]
Can i generate IObservable instead of Task? Thanks

**Comment by christianhelle** [comment]
@janfolbrecht it's not something Refitter can do as it is right now. According to the [README](https://github.com/reactiveui/refit?tab=readme-ov-file#retrieving-the-response), Refit supports `IObservable`  so someone just needs to build that

**Comment by christianhelle** [comment]
@janfolbrecht This could be interesting to try out. I think I'll add support for it in the next few days. You are always more than welcome to implement this yourself and I will be more than happy to approve any pull request with working features

**Reply by christianhelle** [reply, marked as answer]
I just realized after writing this comment that you had already created a pull request #322
